src/pages/stacks-blockchain/stacking.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Understand Stacking
+description: Introduction to the reward mechanism of Proof-of-Transfer
+images:
+  sm: /images/pages/stacking-rounded.svg
+---
+
+## Introduction
+
+Stacking rewards Stacks token holders with bitcoins for providing a valuable service to the network by locking up their tokens for a certain time.
+
+Stacking is a built-in action, required by the "proof-of-transfer" (PoX) mechanism. The PoX mechanism is executed by every miner on the Stacks 2.0 network.
+
+-> Fun fact: The Stacking consensus algorithm is implemented as a smart contract, using [Clarity](/smart-contracts/overview).
+
+## Stacking flow
+
+The Stacking mechanism can be presented as a flow of actions:
+
+![stacking flow](/images/stacking-illustration.png)
+
+1. Make API calls to get details about the upcoming reward cycle
+2. For a specific Stacks account, confirm eligibility (minimum Stacks tokens available and more)
+3. Confirm the BTC reward address and the lockup duration
+4. The transaction is broadcasted and the Stacks tokens will be locked-up
+5. The Stacking mechanism executes reward cycles and sends out rewards to the set BTC reward address
+6. During the lockup period, details about unlocking timing, rewards and more can be obtained
+7. Once the lockup period is passed, the tokens are released and accessible again
+8. Display reward history, including details like earnings for previous reward cycles
+
+If you would like to implement this flow in your own wallet, exchange, or any other application, please have a look at this tutorial:
+
+[@page-reference | inline]
+| /stacks-blockchain/integrate-stacking
+
+## PoX mining
+
+PoX mining is a modification of Proof-of-Burn (PoB) mining, where instead of destroying the committed Bitcoin, it is transferred to eligible Stacks (STX) holders that participate in the Stacking protocol.
+
+-> A PoX miner can only receive newly-minted Stacks (STX) tokens when they transfer bitcoins to eligible owners of Stacks tokens
+
+Miners have to run a software (mining client, aka "miner") on their machines to participate in the PoX mechanism. The mining client implements the PoX mechanism, which ensures proper handling and incentives through four key phases:
+
+- Registration: Miners register for a future election by sending consensus data to the network
+- Commitment: Registered miners transfer bitcoins to participate in the election
+- Election: A verifiable random function chooses one miner to write a new block on the Stacks blockchain
+- Assembly: The elected miner writes the new block and collects rewards in form of new Stacks tokens
+- Distribution: Committed bitcoins are sent to a set eligible Stacks tokens holders
```

Comment:
Two nits: (1) these are 5 phases, but the previous paragraph says "4" (2) "distribution" occurs at the same time as "commitment": when a miner performs their commitment, the BTC is transferred directly.

Reply:
thanks, also updated!